File: backend/API/Application/Features/Shelter/Breeds/Dtos/BreedFilterDto.cs
namespace API.Application.Features.Shelter.Breeds.Dtos
{
    public class BreedFilterDto
    {
        public int Page { get; set; } = 1;

        public int PageSize { get; set; } = 10;

        public string? Search { get; set; }

        public string? SpeciesId { get; set; }
        public string? Sort { get; set; }
    }
}

File: frontend/src/core/application/features/shelter/breeds/dtos/breed-filter-dto.ts
import { FilterItem } from "@/store/use-filter-plp-store/interfaces/filter-item.interface";

export interface BreedFilterDto {
  page: number;
  pageSize: number;
  search?: string;
  storeFilters?: FilterItem[];
  activeFeatures?: string[];
  speciesId?: string;
}